02-primitives/02-strings/.README.md

```diff
@@ -0,0 +1,3 @@
+# Strings - Change subnet number in the given CIDR address
+
+Write a small function, and change the subnet number of the {{index . "cidr"}} CIDR address to {{index . "subnet"}} in the file `exercise.go` near the placeholder `// INSERT YOUR CODE HERE`. Do not use additional variables, or modify the existing code.
```

Comment:
Can we add a hint like "use `strings.Split`  to split strings on an arbitrary delimiter" or similar? Also, I'm afraid people will not get this immediately: can you explain what a CIDR prefix is and what we mean by "changing the subnet number"? I know people should know this already, but let's make this self-contained.

Reply:
Sure, I will include one or two sentences about CIDR.